ProjectSettings/ProjectSettings.asset

```diff
@@ -134,14 +134,8 @@ PlayerSettings:
     16:10: 1
     16:9: 1
     Others: 1
-  bundleVersion: 3
-  preloadedAssets:
-  - {fileID: 0}
-  - {fileID: 0}
-  - {fileID: 0}
-  - {fileID: 0}
-  - {fileID: 0}
-  - {fileID: 0}
+  bundleVersion: 4.1.0
```

Comment:
This is to signify the change in build version under project settings. Previously it was version 3.0.0 but needed to be updated.